### ISSUE TITLE
Incorrect Documentation

### ISSUE THREAD

**Issue by RyanThomas73** [MicrosoftDocs/appcenter-docs#486]
The documentation item 3.2. states that the appcenter binaries can be downloaded directly and links to the releases page of the github repo. The downloaded releases zip, however, does not contain the compiled frameworks that the documentation references. It contains only the source .xcodeproj / .xcworkspace project files. The release zip needs to contain the .framework build results.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 45228922-4a96-8c08-7d27-69ef98d44943
* Version Independent ID: d04ef359-7b17-8a14-970b-5c48820976cd
* Content: [Get Started with iOS - Visual Studio App Center](https://docs.microsoft.com/en-us/appcenter/sdk/getting-started/ios#feedback)
* Content Source: [docs/sdk/getting-started/ios.md](https://github.com/MicrosoftDocs/appcenter-docs/blob/live/docs/sdk/getting-started/ios.md)
* Service: **vs-appcenter**
* GitHub Login: @elamalani
* Microsoft Alias: **emalani**

**Comment by jwargo** [comment]
@RyanThomas73 I've asked the SDK team to respond.

**Comment by RyanThomas73** [comment]
@jwargo Thanks. After adding the comment on the documentation I realized that all of the release zips up to the most recent have the correct files (e.g `1.13.1` has the correct .framework build files in the expected folder structure). It is only the most recent `1.13.2` release zip that does not have the build files, it instead contains the source contents of the github repo.

I entered this issue on the SDK github repo noting the problem with the zip file contents:
https://github.com/Microsoft/AppCenter-SDK-Apple/issues/1349
